include/deal.II/lac/relaxation_block.h

```diff
@@ -147,10 +147,23 @@ class RelaxationBlock :
      * </ol>
      */
     std::vector<std::vector<unsigned int> > order;
+
     /**
      * Return the memory allocated in this object.
      */
     std::size_t memory_consumption() const;
+
+    /**
+     * Temporary ghost vector that is used in the Relaxation method when
+     * performing parallel MPI computations. The user is required to have this
+     * point to an initialized vector that contains at least the locally
+     * active level DoFs.
```

Comment:
It would also be nice to update the constructor of this class to take as arguments *all* of the member variables of the class. But I see that it is already not up to date in this regard.